docs/developer-docs/latest/setup-deployment-guides/configurations/databases/mongodb.md

```diff
@@ -289,25 +289,71 @@ module.exports = ({ env }) => ({
       connector: 'mongoose',
       settings: {
         uri: env('DATABASE_URI'),
+        srv: env.bool('DATABASE_SRV', true),
+        port: env.int('DATABASE_PORT', 27017),
+        database: env('DATABASE_NAME'),
+      },
+      options: {
+        authenticationDatabase: env('AUTHENTICATION_DATABASE', null),
+        ssl: env.bool('DATABASE_SSL', true),
+      },
+    },
+  },
+})
+
+```
+
+`Path: .env`
+
+```
+DATABASE_URI=mongodb+srv://username:<password>@cluster0.blah.mongodb.net/myFirstDatabase?retryWrites=true&w=majority
+DATABASE_NAME=myFirstDatabase
+```
+
+
+::: 
+
+::: tab Combined Method
+
+This method connects via passing individual connection settings.
+
+```js
+module.exports = ({ env }) => ({
+  defaultConnection: 'default',
+  connections: {
+    default: {
+      connector: 'mongoose',
+      settings: {
+        host: env('DATABASE_HOST'),
+        srv: env.bool('DATABASE_SRV', true),
+        port: env.int('DATABASE_PORT', 27017),
+        database: env('DATABASE_NAME'),
+        username: env('DATABASE_USERNAME'),
+        password: env('DATABASE_PASSWORD'),
       },
       options: {
-        ssl: true,
+        authenticationDatabase: env('AUTHENTICATION_DATABASE', null),
+        ssl: env.bool('DATABASE_SSL', true),
       },
     },
   },
-});
+})
+
 ```
 
 `Path: .env`
 
 ```
-DATABASE_URI=mongodb://paulbocuse:<password>@strapidatabase-shard-00-00-fxxx6c.mongodb.net:27017,strapidatabase-shard-00-01-fxxxc.mongodb.net:27017,strapidatabase-shard-00-02-fxxxc.mongodb.net:27017/test?ssl=true&replicaSet=strapidatabase-shard-0&authSource=admin&retryWrites=true&w=majority
+DATABASE_HOST=cluster0.blah.mongodb.net
+DATABASE_NAME=myFirstDatabase
+DATABASE_USERNAME=admin
+DATABASE_PASSWORD=yourpassword
 ```
 
-::: warning NOTE
-The above configuration will create a database called `strapi`, the _default database_ Strapi sets for any **MongoDB** database. If you would like to name your database something else, add the following **key:value pair** into your **settings:** located in your `database.js` file.
 
-`database: 'my-database-name'`
+:::
+
+::::
 
 :::
```

Comment:
```suggestion

```

Please remove this line ;-) or it gets printed:
![Screenshot 2021-03-31 at 17 33 17](https://user-images.githubusercontent.com/4233866/113170889-43a2ad00-9247-11eb-927c-0d8b336a549b.png)

(FYI, you already closed the tabs markup with `::::` on line 356, and the `::: warning NOTE` at line 358 already has its 3 starting colons `:::` :-) )

Reply:
Ah, yes! Gotcha. Have just made the edit now directly to that file. I got confused trying to make the change locally and push to the right place so just edited it directly on here.  Still learning the ropes on how PRs work, I'm usually just working away in my own little bubble!

Reply:
No worries! We are all learning something new each & every day 🤗